src/chk/chk_internal.h

```diff
@@ -248,6 +248,15 @@ struct chk_property {
 	uint32_t			cp_rank_nr;
 };
 
+struct chk_iv {
+	uint64_t		 ci_gen;
+	d_rank_t		 ci_rank;
```

Comment:
just curious what is ci_rank here? master rank?  you can get it from ns?

Reply:
It is some non-master rank from the leaf.

Reply:
from leaf to root? then why update it in chk_iv_ent_refresh.